packages/commons/src/email-manager/emailManager.ts

```diff
@@ -27,8 +28,9 @@ export function initEmailManager(config: EmailManagerConfig): EmailManager {
         from,
```

Comment:
You could also use here `config.emailManagerSender` directly, so that you don't have to do `getSender()` and then pass it back as a parameter when calling `send`

Reply:
I did like that because maybe it will be used with a different sender in some context. What do you think?

Reply:
We could extend it for that when needed I would say - in that case, maybe I would rather have the config variable not as part of the Email Manager config but as part of the config of the service that uses it, and just pass it as param from the outside